closure/compiler/closure_js_library.bzl

```diff
@@ -68,6 +68,25 @@ def _impl(ctx):
     args += ["--expect_failure"]
   if ctx.attr.internal_expect_warnings:
     args += ["--expect_warnings"]
+
+  files = [ctx.outputs.provided, ctx.outputs.stderr]
+  if ctx.attr.style and len(ctx.files.srcs) > 0:
+    files.append(ctx.outputs.jsstyle)
+    style_args = [ctx.executable._clangformat.path,
+                  ctx.outputs.jsstyle.path]
+    style_inputs = [ctx.executable._clangformat]
+    for direct_src in ctx.files.srcs:
+      style_args += [direct_src.path]
+      style_inputs.append(direct_src)
+    ctx.action(
```

Comment:
I'm only talking about using the `convention` and `suppress` attributes on `closure_js_library` to configure the clang style enforcement. I think the convention using the `// clang-format off` comments should stay the same, rather than using `@suppress`.

Also since we're already having design discussion here about enforcement via closure_js_library, I would recommend creating a separate PR for the simple task of adding the formatter to the codebase. That's something we can get merged in quicker.